Makefile

```diff
@@ -6,16 +6,17 @@ GOTOOLS = github.com/golang/lint/golint \
 # Temporary patch to avoid build failing because of the outdated documentation example
 # TODO: uncomment below commented lines and remove any line that uses $(NODOCS)
 NODOCS = $(shell go list ./... | grep -v /docs/)
+PKGS = $(shell go list ./... | egrep -v "\/docs\/|jmx")
 
-all: validate test
+all: lint test
 
 deps: tools
 #	@go get -v -d -t ./...
-	@go get -v -d -t $(NODOCS) #todo:remove
+	@go get -v -d -t $(NODOCS) #TODO: remove
 
 test: deps
-#	@gocov test ./... | gocov-xml > coverage.xml
-	@gocov test $(NODOCS) | gocov-xml > coverage.xml
+	@gocov test github.com/newrelic/infra-integrations-sdk/jmx > /dev/null # TODO: fix race for jmx package
+	@gocov test -race $(PKGS) | gocov-xml >> coverage.xml
```

Comment:
Do we want to test with `-race` by default? Is just a question

Reply:
Yes, we want to support concurrency